test/test_product_info.py

```diff
@@ -30,4 +30,32 @@ def test_handler_stockpear(self, classTest):
         expectedOutput = "{} are out of stock.".format(MOCK_PRODUCT_DATA[3]["names"].capitalize())
         assert(classTest.handle(message) == expectedOutput)
 
+@pytest.mark.prod_info
+class TestNutritionInfo:
```

Comment:
Should add a case where the handler fails to ensure it does not give answers for nothing.

Reply:
@SanaBob